rubin_sim/maf/metrics/incremental_template_metric.py

```diff
@@ -22,12 +23,14 @@ class TemplateTime(BaseMetric):
         Default 3.
     seeing_percentile : `float`, opt
         Maximum percentile seeing to allow in the qualified images (0 - 100).
-
-        Default 50.
+        XXX--deceptive kwarg since it checks the percentile at each point on the sky
```

Comment:
I really don't know if per-point-on-the-sky was the intended plan or not. I kind of think it was, so not necessarily deceptive but could be clarified (it also reads backwards to me) 
"Max percentile of seeing values (at each RA/Dec) to allow in the template source images (0-100)". 
It's also worth checking with Eric if the default should be 50 or 100. 
Previously we had a hard cutoff, which wouldn't be the worst way to go either (could potentially have both kwargs and use one to override the other).

Reply:
Pinging @ebellm for feedback on the update to the default value.

Reply:
Doing a percentile cut per-point-on-sky seems very wrong to me since then you could have 5 images with fantastic seeing, but it'll say it can't make a template because their aren't 3 over 50th percentile.

Reply:
I can see that .. and vice-versa if you have a spot on the sky with only terrible seeing, you might want to make a template anyway. 
I wonder if the actual way to do it is to mix the two together - if you have X images better than some (high percentile over the whole sky) value, make the template and if you have X images better than the 50% level at this point in the sky, make the template anyway as well.

Reply:
You are both right, this is tricky--you always want the best images, but what counts as "good enough" does have some dependence on the minimum airmass you can observe the field at, for instance.  I had though to use percentiles to avoid undue assumptions about the shape of the seeing distribution, but a hard cut to avoid the worst seeing tail + a local percentile might be a decent approach?

Reply:
Sounds fine to me.